src/copy-paste.js

```diff
@@ -20,7 +20,9 @@ define([
             width: 0,
             height: 0
         }).css(offScreen).appendTo('body'),
-        isMac = /Mac/.test(navigator.platform);
+        isMac = /Mac/.test(navigator.platform),
+        isChrome = /Chrome/.test(navigator.userAgent),
+        isSafari = /Safari/.test(navigator.userAgent) && !isChrome;
```

Comment:
So these are userAgent strings that contain both "Safari" and "Chrome", and they're really Chrome?